### PR TITLE
Add a pipeline for Linux ARM64 python package

### PR DESCRIPTION
**Pull request by snnn**
**Description**:

1. Support cross-compiling the python package for ARM. Validated with the latest linaro toolchain: gcc--7.5.0-2019.12 for ARM64.
2. Add a pipeline for it.

If we want to support more compiler versions, like the 4.8.3 used by raspberry pi, we need to suppress more warnings. It's easy to do. But so far we don't have such requirement, so I discarded these changes from this PR. 

I'll add a doc later.

**Motivation and Context**
- Why is this change required? What problem does it solve?
- If it fixes an open issue, please link to the issue here.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -62,9 +62,23 @@
...
+#PEP 599 defines the following platform tags:
+# manylinux2014_x86_64
+# manylinux2014_i686
+# manylinux2014_aarch64
+# manylinux2014_armv7l
+# manylinux2014_ppc64
+# manylinux2014_ppc64le
+# manylinux2014_s390x
+
+manylinux_tags = ['manylinux1_x86_64', 'manylinux1_i686', 'manylinux2010_x86_64', 'manylinux2010_i686', 'manylinux2014_x86_64', 'manylinux2014_i686', 'manylinux2014_aarch64', 'manylinux2014_armv7l', 'manylinux2014_ppc64', 'manylinux2014_ppc64le', 'manylinux2014_s390x']
```
Comment (prabhat00155) [inline setup.py:76]: Define list this way:
```
manylinux_tags = [
    'manylinux1_x86_64',
    'manylinux1_i686',
    ...,
]
```

File: setup.py [hunk 2]
```diff
@@ -62,9 +62,23 @@
...
+ENV_AUDITWHEEL_PLAT = environ.get('AUDITWHEEL_PLAT', None)
+if ENV_AUDITWHEEL_PLAT in manylinux_tags:
     is_manylinux1 = True
 
+print("is_manylinux1=%s" % is_manylinux1)
```
Comment (prabhat00155) [inline setup.py:81]: Is this for debugging?

File: setup.py [hunk 3]
```diff
@@ -116,6 +130,7 @@ def run(self):
                 self._rewrite_ld_preload(to_preload)
             _bdist_wheel.run(self)
             if is_manylinux1:
+                #The following part doesn't work in cross-compiling
```
Comment (prabhat00155) [inline setup.py:133]: Space after # missing.

File: setup.py (hunk 2 above)
Comment (snnn) [reply setup.py:81]: Yes.

File: setup.py (hunk 1 above)
Comment (snnn) [reply setup.py]: Fixed.

File: setup.py (hunk 3 above)
Comment (snnn) [reply setup.py]: Fixed.

File: setup.py (hunk 2 above)
Comment (prabhat00155) [reply setup.py:97]: Should we be checking this in then?

File: tools/ci_build/github/azure-pipelines/arm.yml [hunk 4]
```diff
@@ -0,0 +1,44 @@
+jobs:
+- job: arm
+  timeoutInMinutes:  60
```
Comment (prabhat00155) [inline tools/ci_build/github/azure-pipelines/arm.yml:3]: Looks like an extra space before 60.

**Comment by snnn** [comment]
Get conflict with master again.  

Resolved.

**Pull request closed without merging**